isis/src/base/apps/handmos/handmos.xml

```diff
@@ -47,19 +47,41 @@
-      the track-band.</b>
+      The TRACK feature creates a separate tracking cube in addition to the mosaic cube, and
+      contains information for the source files of every pixel within the output mosaic.
+      This cube will have the same base-name as the mosaic cube, but will end in 
+      "_tracking.cub". The tracking cube must always reside in the same directory as the
+      mosaic cube to be properly accessed; this means that if the mosaic cube is copied or
+      moved, then its associated tracking cube must be copied or moved to the same location.
+      <b>The tracking cube will always be of type unsigned integer. Depending on the
+      bit-type of the mosaic cube and/or the number of bands it contains, the tracking cube
+      may be as much as four times the size of the mosaic cube itself.</b>
     </p>
     <p>
-      The Track feature works with Priority options ONTOP (with HIGHSATURATION and
-      LOWSATURATION parameters) and BENEATH for single band input cubes.  It does NOT support
-      multiple band input cubes with these priority options. It does work for multiband cubes
-      when PRIORITY=BAND. Furthermore, this feature is NOT supported when PRIORITY=AVERAGE.
+      The tracking cube can be used appropriately through the QVIEW-AdvancedTracking
+      tool. As the user pans across the displayed mosaic, for every mosaic pixel location,
+      QVIEW-AdvancedTracking will interactively report the index, the filename and the
+      serial number of the input cube that was input to handmos for that specific pixel
+      location. Since the tracking cube is of bit-type unsigned integer, the DN values of
+      0, 1 and 2 are reserved for NULL, LRS and LIS, respectively, so valid pixel DN values
+      will begin at an offset of 3. In other words, a pixel of DN value 3 in the tracking
+      cube means that this same pixel within the mosaic was taken from the first input
+      image. The tracking cube cannot be used outside of the QVIEW-AdvancedTracking tool
+      except as a visual representation of the source cubes for the different pixels. 
+    </p>
+    <p>
+      The TRACK feature works with Priority options ONTOP and BENEATH for single band input
+      cubes. It works for multiband cubes for PRIORITY=ONTOP only when the NULL,
+      HIGHSATURATION and LOWSATURATION options are set to true. It also works for multiband
+      cubes when PRIORITY=BAND. Furthermore, this feature is NOT supported when
+      PRIORITY=AVERAGE.
+    </p>
+    <p>
+      <b>Please Note: Older versions of handmos (prior to isis3.5.3) no longer work with
+      mosaics that contain an attached tracking band. Mosaics including an attached tracking
+      cube can be updated and reformatted by running the program trackextract. This new
+      utility will create a new version of the mosaic with the appropriate label information
+      and a separate tracking cube to which it points.</b>

```

Comment:
this paragraph doesn't make sense to me. It seems like this has attached tracking cube and attached tracking band mixed up. The tracking cube is the new way, the tracking band is the old way. Also is this talking about the fact that old versions of handmos don't see the tracking cube or that new versions of handmos don't work with the tracking band? Both are compatibility issues, but it seems like they are getting mixed up here.